gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/EntitiesRequestContext.java

```diff
@@ -6,32 +6,40 @@
 
 public class EntitiesRequestContext extends QueryRequestContext {
   private final String entityType;
+  private final String timestampAttributeId;
```

Comment:
Instead of recomputing the timestamp column for a given entity over and over, we compute once and carry in the context.